Remove first item for the prepare for next class instructions as it's… (#465)
CodeYourFuture/syllabus: docs/contributing/minutes.md

sidebar_label: Team Minutes
---

## 2022-07-21

Attendees: Sally, Apoorva, Greg, Mitch

### ❗Actions from last time

- [ ] Sally: send list of some more people for React module team
- [ ] Ali: chat with Lily about more people for React module team

> - [ ] Ali: coordinate with applicants so far for React module team

Reviewer: I think these can all be marked as done - we have a React module team formed now.

- [ ] Sally: follow up with ZA at some point about why they needed to conduct interviews

### Fundamentals feedback

- Feels like we need a lot of TA support for Fundamentals
- Add something to the module coordinator notes to think about the number of TAs required for a Fundamentals session
- TAs not assigned to a room in Scotland - so they all jump around

> - They need experienced people to guide them when they gooff posted

Reviewer: ```suggestion
- They need experienced people to guide them when they go off posted
```

Not sure if there's a missing word at the end too?

> - LDN9 is a completely new group of volunteers - so good test of our unspoken assumptions

Reviewer: Don't think this needs to go into the notes, but from talking with some LDN8 tech ed volunteers, I think some are planning to help out. I got the impression that they're not aware of explicit timelines around LDN9. And not necessarily volunteering in leadership roles.

- Covering a lot of blocks in the Fundamentals course
- A lot of pushback from trainees, why are we using Blocks - like kids stuff
- Found the Blockly fundamentals homework overwhelming
- Greg: The split up of the homework was arbitrary - so we can rearrange it so we can add some more homework to Week 1 if required, for example
- A section on MVP in Week 2 - not realistic for it to be completed in 10 mins - more like 20 & 20 or 15 & 15
- Students said stuff "user stories" wasn't important
- Fundamentals will give students a better though far from complete impression of what being a software engineer involves
- Come up with a concrete outcome for the coursework - as trainees struggle to follow more abstract instructions
- Trainees are very wary of people in institutions - experience of being lied to etc
- Some things like create a discussion in a slack thread aren't happening

> - Smaller spaces with games and retreat spaces good for breaking up large classes with 50+ peple

Reviewer: ```suggestion
- Smaller spaces with games and retreat spaces good for breaking up large classes with 50+ people
```

- Want to get a reduction in the early course dropout rate - target for next year
- 60+ for fundamentals is too big!
- Need to keep encouraging trainees to ask questions in main channels and _not_ DM TAs
- Asking questions, handling conflict are all essentials skills we need to keep developing in trainees as they progress through the course

### Immersive program

- Far more selective and demanding than the main course
- See link for more detail -> https://docs.google.com/document/d/1hNolfVFbwb3mYML3OaNw1zStACZbw5np-Obpx6eLtJ4/edit
- Course structure here -> https://docs.google.com/document/d/1V6HEu_OcJ3MHH-aHzUfANf06VJa1rPcGHcpBwql7QLA/edit#heading=h.dgaqq1am7dqu
- Invitation has been sent out to trainees
- Program is designed around x5 10 day sprints
- Hardly anyone has applied for the course so far - despite there being no consequences for applying

### Teacher training

- Currently running 2 iterations of the teacher training course in West Midlands and North West. West Midlands -> completely remote, North West -> Hybridg.

> - Current format involves 4/5 x 2 hour sworkshops ( with plenty .of attendee participation and engagement). Using Miro to collect ideas - working well thus far.

Reviewer: ```suggestion
- Currently running 2 iterations of the teacher training course in West Midlands and North West. West Midlands -> completely remote, North West -> Hybrid.
- Current format involves 4/5 x 2 hour workshops ( with plenty .of attendee participation and engagement). Using Miro to collect ideas - working well thus far.
```
